### DISCUSSION TITLE
These projections are a joy to use!

### DISCUSSION THREAD

**Discussion by jnm2** [microsoft/win32metadata#280] [General]
There's no Discussions tab on https://github.com/microsoft/cswin32, but I just wanted to say I'm absolutely loving it. The enums, the handle wrapper structs, being able to pass `null` to SafeHandle parameters, the intellisense documentation, catching mistakes I made as I replace my hand-written definitions... a lot just feels easy and it feels right.

Thanks to everyone who is collaborating on these tools!

/cc @AArnott from CsWin32

**Comment by AArnott** [comment]
I'm very glad to hear you're pleased with it, @jnm2. We still have lots to do to make it even better, but I'm glad we're hitting the mark already for you.
I've activated the [discussions](https://github.com/microsoft/CsWin32/discussions) feature on CsWin32. I'm new to it so we'll see how it goes.
